src/commands/classes/Contest.command.ts

```diff
@@ -108,21 +235,116 @@ export default class ContestCommand extends Command {
   //tally vote and announce winner of Contest
   //top 3 get move to unused entries folder to be reshuffled once all entries have been gone through
   public end(args: Array<string>, message: Message): void{ //
+    if(contestData.contestActive){
+      console.log("---Ending Contest---");
+      let reaction_numbers = ["\u0030\u20E3","\u0031\u20E3","\u0032\u20E3","\u0033\u20E3","\u0034\u20E3","\u0035\u20E3", "\u0036\u20E3","\u0037\u20E3","\u0038\u20E3","\u0039\u20E3"];
+      let files = contestData.entries;
+      console.log("Determining contest channel");
+      let contestChannel = message.guild.channels.cache.find((channel) => channel.name === contestData.contestChannelName) as TextChannel;
+      console.log("Successful");
+      
+      console.log("Determining Contest origin message");
+      contestChannel.messages.fetch(contestData.messageId).then(contestMessage => {
+        //console.log("ContestMessage ID: "+contestMessage.id+"\t\t"+contestMessage.toString());
+        
+        let reactions = contestMessage.reactions.cache;
+        console.log("Successful");
+        let reacEmoji = [];
+        let reacCount = [];
+        for (let reaction of reactions)
+        {
+          reacEmoji.push(reaction[1].emoji);
+          console.log(reaction);
+          reacCount.push(reaction[1].count);
+          console.log("reacCount:  " + reacCount);
+
+        }
+
+        console.log("Determining winners");
+        let winners = [];
+        while(winners.length < 3){
+          let max = 0;
+          for(let x of reacCount){
+            max = x > max ? x : max;
+          }
+          
+          while(reacCount.includes(max)){
+            let winner = {};
+            let x = reacCount.indexOf(max);
```

Comment:
Instead of rechecking reacCount for the max after found. Do the max check and index together. Just go through the indexes of reacCount and save the index with the largest Max count. Also, there might be two reactions with same max count. What happens then?